src/app/tests/suites/certification/Test_TC_S_3_1.yaml

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
+
+name: 132.3.1. [TC-S-3.1] Functionality with DUT as Client
+
+PICS:
+    - S.C
+
+config:
+    nodeId: 0x12344321
+    cluster: "Basic Information"
+    endpoint: 0
+
+tests:
+    - label: "Note"
+      verification: |
+          For DUT as client test cases, Chip-tool command used below are an example to verify the functionality. For certification test, we expect DUT should have a capability or way to run the equivalent command.
+      disabled: true
+
+    - label: "Step 0:"
+      verification: |
+          "Step 0: Preparation:
+          - TH as server exposes a Scenes Management server cluster on Endpoint: EndpointID, supporting all possible commands and corresponding attributes.
+          - Commission DUT to TH
+      disabled: true
+
+    - label: "Step 1: DUT issues a C_ADD_SCENE command to the Test Harness."
```

Comment:
Can you clarify which pieces of this test need to be manual and why?

Reply:
These are client side of tests. Since clients may have their own API to trigger the required action, in the script they provided chip-tool commands as a reference/example. clients should run equivalent commands on their platform to test this functionality.

Reply:
As mentioned above, the only reason this is manual is because they are client side tests